### DISCUSSION TITLE
Contribute sample prompts!

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#203] [General]
Hi everyone, I added a welcome screen for the next version of Alpaca
https://github.com/Jeffser/Alpaca/issues/137

Each time the chat is empty Alpaca pulls 3 random prompts to give to the user so I wanted to ask you guys to give me some serious or funny prompts for the the welcome screen!

Just make sure it's nothing offensive and it makes sense for a conversation starter.

Also, keep it short, 4 or 5 words max.

Also please note that the prompts will only be in English due to many models not understanding other languages, so to keep it consistent it will only be English prompts

![image](https://github.com/user-attachments/assets/720c5d64-ed78-4852-b074-00d88aea3395)

**Comment by olumolu** [comment]
Added a pull request which will merge the questions from my side.
https://github.com/Jeffser/Alpaca/pull/206

**Reply by Jeffser** [reply]
thanks!

**Comment by Jeffser** [comment]
Please just write the prompts here so you don't have to make a pull request

**Comment by loulou64490** [comment]
Here are some prompt!

Write an Haïku about AI
What is the meaning of life ?
Generate a video game with python 
Explain quantum physics in simple terms
Explain the theory of relativity
Explain how photosynthesis works
Recommend a film about nature
What is nostalgia ?

**Reply by olumolu** [reply]
Videogame is not a easy task for smaller ai models like some 2b or 3b and too much code is required i don't think having this as option can help rather it will complicate stuff more
And demo questions like asking for a quantum mechanics or wave function and other core physics stuff is too much.

**Reply by loulou64490** [reply]
model like llama 3.1 or gemma2 can respond easily to all those questions, and they are the most used model... 
when someone use a specific model, he know what question he will ask

**Reply by Jeffser** [reply]
Hi, I added those prompts except the videogame one, it's better to keep it simple so that every model can handle it

**Comment by olumolu** [comment]
From my side some extra

Questions
Explain superconductors to a five years old

Plan a mental health day to help me relax

Write a message
(that goes with a kitten gif for a friend on a rough day)

Make me a personal webpage
(after asking me three/some questions)

Write a thank-you note
(to my interviewer)

Suggest fun activities
(to help me make friends in a new city)

Write a Python script
(to automate sending daily email reports)

Test my knowledge
(on ancient civilizations)

Design a programming game
(teach basics in a fun way)

Explain nostalgia to a kindergartener

**Reply by olumolu** [reply]
@Jeffser have you added them if you like to.

**Reply by Jeffser** [reply]
I'm not sure if I like most of them
"Make a personal webpage" > too big and vague, not a conversation starter
"Write a thank-you note" > the model won't know what to do, needs more context
"Write a python script" > too vague
"Test my knowledge" > also too vague
"design a programming game" > too big
"explain nostalgia to a kindergartener"  > too similar to "what is nostalgia?"

**Reply by olumolu** [reply]
Yes this are not that good as my initials so i put this here.
Still superconducter is something which is good to added.

**Reply by Jeffser** [reply]
yeah sure, that one is good

**Comment by ProjectMoon** [comment]
Just yoink the ones from Open-WebUI (which I'm pretty sure were just yoinked from ChatGPT).

**Reply by Jeffser** [reply]
I could do that but I wanted to give people the opportunity of seeing their own prompts on the app
